trackier-ios-sdk/Classes/TrackierSDKInstance.swift

```diff
@@ -87,6 +87,11 @@ class TrackierSDKInstance {
     
     private func makeWorkRequest(kind: String) -> TrackierWorkRequest {
         let wrk = TrackierWorkRequest(kind: kind, appToken: self.appToken, mode: self.config.env)
+        if (self.config.getSDKType() != "ios") {
+                   DeviceInfo.sdkVersion = self.config.getSDKVersion()
```

Comment:
indentation @IloveJavaa

Reply:
we need to send both of these information to the backend which you are not currently doing